prometheus/exporter/watchdog.py

```diff
@@ -25,7 +25,9 @@
 import time
 import common
 
+loggerRoot = logging.getLogger()  
 logger = logging.getLogger("watchdog")  
+loggerCommon = logging.getLogger("common")  
```

Comment:
Console displays "No handlers could be found for logger "common"", why we need to deal with "common"?

Reply:
use pai/pai-management/k8sPaiLibrary/maintainlib/common.py   ssh method. common.py also call logging lib and will generate "No handlers could be found for logger "common""

Reply:
If not deal with it, it log will also up to root logger and keep generate to console